Dockerfile.cosmovisor

```diff
@@ -38,8 +38,8 @@ ADD https://github.com/cosmos/cosmos-sdk/releases/download/cosmovisor/${COSMOVIS
 RUN tar -xvf /tmp/cosmovisor.tar.gz -C /usr/local/bin/ && \
     chmod +x /usr/local/bin/cosmovisor
 
-ARG ALLORAD_CURRENT_VERSION="v0.3.0"
-ARG ALLORAD_UPGRADE_VERSION="v0.4.0"
+ARG ALLORAD_CURRENT_VERSION="v0.4.0"
+ARG ALLORAD_UPGRADE_VERSION="v0.4.1"
```

Comment:
needs to be v0.5.0

Reply:
https://github.com/allora-network/allora-chain/pull/594